src/backend/distributed/citus--8.2-2--8.2-3.sql

```diff
@@ -0,0 +1,27 @@
+/* citus--8.2-2--8.2-3 */
+
+SET search_path = 'pg_catalog';
+
+DROP FUNCTION master_update_node(node_id int,
```

Comment:
We are replacing the function, since the new arguments are optional I dropped this function. The usage should be the same as before, the only downside would be if a user has granted extra roles with the privilege to invoke this function, those changes are lost.

Wonder if it is expected that someone would grant other roles with this permission and we should prevent the grants to be lost on update.

Reply:
Good question. We routinely drop functions like master_add_node, so I think we can continue doing it here. 

We once took the position that these functions should not explicitly require superuser, because it would be more secure to call them only via an unprivileged role, though is difficult to maintain in practice. I think we should perhaps maintain such a role ourselves. That role can then be granted to a user.

Reply:
Like the idea of the citus managed role, will not implement it now though